alternator/serialization.hh

```diff
@@ -62,6 +62,10 @@ big_decimal unwrap_number(const rjson::value& v, std::string_view diagnostic);
 // when the given v does not encode a number.
 std::optional<big_decimal> try_unwrap_number(const rjson::value& v);
 
+// unwrap_bytes decodes byte value, it raises ValidationException if value comes from query or
```

Comment:
Actually I see above api_error::serialization. That's not the same thing as ValidationException, right?

Reply:
yes, thanks, forgot to update after I made the code to distinguish between those two as dynamo does